### ISSUE TITLE
Issue with DTCR_WF and DTCR_SS Get_Train_Valid_Test

### ISSUE THREAD

**Issue by MittyHills** [sidhomj/DeepTCR#86]
Sorry if theres an obvious answer to this, I'm not very experienced with python and am learning for my Honours degree. 

When running this part of the DTCR_WF script:

DTCR_WF.Get_Train_Valid_Test(test_size=0.25)
DTCR_WF.Train()

I'm getting an error that reads:

"                                       line 4014, in
Get_Train_Valid_Test
    raise Exception('Choose different train/valid/test parameters!')

Exception: Choose different train/valid/test parameters!  "

I also am having what feels like a related issue with DTCR_SS using the same part of it's script, however it does not throw an error and commences training, however nothing occurs in the training and it seems to continue until stopped with the training loss, validation loss, testing loss, etc..., all reading 0. 

Again, sorry if there is an obvious answer. I'd appreciate any help. Thankyou. I should note that the data I'm using works fine using the unsupervised script.

**Comment by MittyHills** [comment]
Nevermind, I've figured it out!
